packages/core/utils/src/replaceBundleReferences.js

```diff
@@ -32,117 +32,110 @@ export async function replaceBundleReferences({
   bundleGraph,
   contents,
   map,
-  getInlineReplacement,
-  getInlineBundleContents,
+  replaceInline,
+  replaceUrls = true,
+  relative = true,
 }: {|
   bundle: Bundle,
   bundleGraph: BundleGraph,
   contents: string,
-  getInlineReplacement: (
-    Dependency,
-    ?'string',
-    string,
-  ) => {|from: string, to: string|},
-  getInlineBundleContents: (
-    Bundle,
-    BundleGraph,
-  ) => Async<{|contents: Blob, map: ?(Readable | string)|}>,
+  relative?: boolean,
+  replaceInline?: {|
+    getInlineReplacement: (
+      Dependency,
+      ?'string',
+      string,
+    ) => {|from: string, to: string|},
+    getInlineBundleContents: (
+      Bundle,
+      BundleGraph,
+    ) => Async<{|contents: Blob, map: ?(Readable | string)|}>,
+  |},
+  replaceUrls?: boolean,
   map?: ?SourceMap,
 |}): Promise<BundleResult> {
   let replacements = new Map();
 
-  for (let {
-    dependency,
-    bundleGroup,
-  } of bundleGraph.getBundleGroupsReferencedByBundle(bundle)) {
+  for (let dependency of bundleGraph.getExternalDependencies(bundle)) {
```

Comment:
@devongovett what do you think of this api change?

Reply:
I think I like it? Is it pretty much just splitting `getBundleGroupsReferencedByBundle` into two parts?

Reply:
Kind of. Except dependencies that don't resolve to anything are also included here.

In pretty much every usecase the dependency is resolved, so maybe the former tuple form makes sense, but despite that I like having them separate.